### PR TITLE
#1943 adding in code for hosting multiple websites in one virtual host

### PR DESCRIPTION
**Pull request by camdixon**
## This PR is a:
[ ] New topic
[ x ] Content fix or rewrite
[ x ] Bug fix or improvement
 

## Summary
 
When this pull request is merged, it will show users how to simplify Nginx configs to host multiple Magento websites and store views with one virtual host file.  It allows to that Nginx configuration to stay much cleaner, and more maintainable.

whatsnew
[These changes](https://devdocs.magento.com/guides/v2.1/config-guide/multi-site/ms_nginx.html) show users how to simplify Nginx configs to host multiple Magento websites and store views with one virtual host file. It allows to that Nginx configuration to stay much cleaner, and more maintainable.

### CODE REVIEW

**Comment by magento-cicd2** [comment]
[CLA assistant check](https://cla.dev.magento.com/magento/devdocs?pullRequest=1953) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dev.magento.com/magento/devdocs?pullRequest=1953) before we can accept your contribution.<br/>

**Comment by camdixon** [comment]
I have tried with both Chrome and Firefox to sign the CLA.  It does not work.

**Comment by camdixon** [comment]
@shrielenee how is this going?  Do I need updates?

**Comment by shrielenee** [comment]
@kandy Can you please review these technical changes? @camdixon has implemented a solution to simplify Nginx configs to host multiple Magento websites and store views with one virtual host file using maps. See also GH Issue #1943 for more information.

**Comment by shrielenee** [comment]
Hey @camdixon - Your technical changes have been approved and I've performed a light copy edit on the content. Can you get the CLA signed? Thanks!

File: guides/v2.1/config-guide/multi-site/ms_nginx.md [hunk 1]
```diff
@@ -17,30 +17,31 @@ functional_areas:
...
 Refer to [Create websites]({{ page.baseurl }}/config-guide/multi-site/ms_websites.html#step-2-create-websites) and [Create store views]({{ page.baseurl }}/config-guide/multi-site/ms_websites.html#step-4-create-store-views) for help locating these values.
     </div>
 
 ### Roadmap for setting up multiple websites with nginx
-Setting up multiple stores consists of the following tasks:
+To set up multiple stores:
 
 1.	[Set up websites, stores, and store views]({{ page.baseurl }}/config-guide/multi-site/ms_websites.html) in the {% glossarytooltip 18b930cf-09cc-47c9-a5e5-905f86c43f81 %}Magento Admin{% endglossarytooltip %}.
-2.	Create one [nginx virtual host](#ms-nginx-vhosts) per Magento {% glossarytooltip a3c8f20f-b067-414e-9781-06378c193155 %}website{% endglossarytooltip %} (steps detailed below).
+2.	Create a [nginx virtual host](#ms-nginx-vhosts) to map many websites or one [nginx virtual host](#ms-nginx-vhosts) per Magento {% glossarytooltip a3c8f20f-b067-414e-9781-06378c193155 %}website{% endglossarytooltip %} (steps detailed below).
```
Comment (bdenham) [inline guides/v2.1/config-guide/multi-site/ms_nginx.md]: Create **an** ...

File: guides/v2.1/config-guide/multi-site/ms_nginx.md [hunk 2]
```diff
@@ -51,11 +52,54 @@ Setting up multiple stores consists of the following tasks:
     *   `$MAGE_RUN_CODE` is the unique website or store view code that corresponds to `$MAGE_RUN_TYPE`.
 
 ## Step 2: Create nginx virtual hosts {#ms-nginx-vhosts}
-This section discusses how to load websites on the {% glossarytooltip 1a70d3ac-6bd9-475a-8937-5f80ca785c14 %}storefront{% endglossarytooltip %}. You can use either websites or store views; if you use store views, you must adjust parameter values accordingly. You must complete the tasks in this section as a user with `root` privileges.
+This section discusses how to load websites on the {% glossarytooltip 1a70d3ac-6bd9-475a-8937-5f80ca785c14 %}storefront{% endglossarytooltip %}. You can use either websites or store views; if you use store views, you must adjust parameter values accordingly. You must complete the tasks in this section as a user with at least `sudo` privileges.
```
Comment (bdenham) [inline guides/v2.1/config-guide/multi-site/ms_nginx.md]: ~~at least~~

File: guides/v2.1/config-guide/multi-site/ms_nginx.md [hunk 3]
```diff
@@ -51,11 +52,54 @@ Setting up multiple stores consists of the following tasks:
...
 
-{% collapsible To create virtual hosts: %}
+You can use one [nginx virtual host](#ms-nginx-vhosts) or many [nginx virtual host](#ms-nginx-vhosts) to load multiple websites. Using just one virtual host file allows you too keep your [nginx virtual host](#ms-nginx-vhosts) file simple and clean, and using many [nginx virtual host](#ms-nginx-vhosts) allows customizations for each particular store, such as a custom location for `french.mysite.mg`.
```
Comment (bdenham) [inline guides/v2.1/config-guide/multi-site/ms_nginx.md]: Consider a more imperative mood for the last two sentences, something like: "By using just one virtual host file, you can keep your [nginx virtual host](#ms-nginx-vhosts) file simple and clean. By using many [nginx virtual hosts](#ms-nginx-vhosts), you can customize each store, such as a custom location for `french.mysite.mg`

File: guides/v2.1/config-guide/multi-site/ms_nginx.md [hunk 4]
```diff
@@ -51,11 +52,54 @@ Setting up multiple stores consists of the following tasks:
...
+		nginx: configuration file /etc/nginx/nginx.conf test is successful
+
+	If errors display, check the syntax of your virtual host configuration files.
+
+5.	Create symbolic link in the `/etc/nginx/sites-enabled` directory:
+
+		cd /etc/nginx/sites-enabled
+		ln -s /etc/nginx/sites-available/magento magento
+
+For more detail about the map directive, see [nginx documentation on the map directive](http://nginx.org/en/docs/http/ngx_http_map_module.html#map){:target="_blank"}.
```
Comment (bdenham) [inline guides/v2.1/config-guide/multi-site/ms_nginx.md]: For more details (plural)

Comment (camdixon) [reply guides/v2.1/config-guide/multi-site/ms_nginx.md]: @shrielenee it should show CLA as signed now.

**Comment by camdixon** [comment]
@shrielenee it should show CLA as signed now.

File: guides/v2.1/config-guide/multi-site/ms_nginx.md (hunk 1 above)
Comment (shrielenee) [reply guides/v2.1/config-guide/multi-site/ms_nginx.md]: 👍

File: guides/v2.1/config-guide/multi-site/ms_nginx.md (hunk 2 above)
Comment (shrielenee) [reply guides/v2.1/config-guide/multi-site/ms_nginx.md]: 👍

File: guides/v2.1/config-guide/multi-site/ms_nginx.md (hunk 3 above)
Comment (shrielenee) [reply guides/v2.1/config-guide/multi-site/ms_nginx.md]: 👍

**Comment by shrielenee** [comment]
Thanks @camdixon for this worthy addition! Merged in.

**Pull request merged**